src/margo-api-reference/workload-api/application-package-api/application-description.linkml.yaml

```
@@ -201,10 +201,155 @@ classes:
   DeploymentProfile:
     description: Represents a deployment configuration for the application.
     rank: 60
+    attributes:
+      id:
+        description: >-
+          An identifier for the deployment profile, given by the application developer, used to uniquely identify this deployment profile in scope of this application description. 
+          The id can be used to display deployment profiles in a user interface and being able to unique distinguish each one of them.
+        rank: 10
+        range: string
+        required: true
+      description:
+        description: >-
+          This human-readable description of a deployment profile allows to provide additional context about the deployment profile. E.g., the application developer can use this to provide information about the deployment profile's purpose,
+          such as the intended use case. Additionally, the application developer can use this to provide further details on resources, peripherals and interfaces required to run the application.
+        rank: 20
+        range: string
+        required: false
+      requiredResources:
+        description: >-
+          Required resources element specifying the resources required to install the application. 
+          See the [Required Resources](#requiredresources-attributes) section below.
+        rank: 30
+        range: RequiredResources
+        required: false
```

```suggestion
      requiredResources:
        description: >-
          Required resources element specifying the resources required to install the application. 
          See the [Required Resources](#requiredresources-attributes) section below.
        rank: 30
        range: RequiredResources
        required: false
        multivalued: true
        inlined: true
        inlined_as_list: true
```

Replaces [this comment](https://github.com/margo/specification/pull/85/files#r2086183421). If you agree, simply apply the suggestion, otherwise don't apply it.

---

I have now made the supported cpu architectures an array. I'm not sure why we would want to make the entire `requiredResources` element a list.